packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -358,7 +406,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
     @include mdc-theme-prop(color, primary);
   }
 
-  &.mdc-textfield--upgraded:not(.mdc-textfield--fullwidth):not(.mdc-textfield--multiline):not(.mdc-textfield--box)::after {
+  &.mdc-textfield--upgraded:not(.mdc-textfield--fullwidth):not(.mdc-textfield--textarea):not(.mdc-textfield--box)::after {
```

Comment:
It should be feasible to remove this section (and the after-styles mixin above); these have been replaced by bottom-line styles. (I think this should also fix #664.)